### PR TITLE
[Cloudnative] Update null checks and if checks in the rename processor

### PR DESCRIPTION
**Pull request by ishleenk17**
- Bug

## What does this PR do?

This PR adds the if check for event.original and the ignore_missing check for the renaming of message to event.original
The packages where the changes are made are as below:

- istio
- nats
- nginx_ingress_controller
- stan

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

### Testing

After adding the check if event.original is null, it takes value of message as shown below: 
Screenshot 2023-09-26 at 3 20 28 PM

If event.original is present. For testing it was set in the pipeline itself as below: 
Screenshot 2023-09-26 at 3 20 39 PM

As a result the event.original has the original value and not the message value. 
Screenshot 2023-09-26 at 3 19 48 PM

## Related issues

- Relates https://github.com/elastic/integrations/issues/7822

### CODE REVIEW

File: packages/istio/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.4.2"
+  changes:
+    - description: Add null check and ignore_missing check to the rename processor
```
Comment (gizas) [inline packages/istio/changelog.yml:4]:
```suggestion
    - description: Add null check to the message rename processor
```

Comment (gizas) [reply packages/istio/changelog.yml:4]: Repeat for all the rest if you agree. 
The `ignore_missing check` dont see it introduced ? Correct?

**Comment by gizas** [comment]
I would expect some test evidence like a grok test or sth that this change worked. Apart from that is ok.

**Review by gizas — APPROVED** [review]

File: packages/istio/changelog.yml (hunk 1 above)
Comment (ishleenk17) [reply packages/istio/changelog.yml:4]: Correct this was added to a couple other integrations changed as part of this bigger changes. But cloudnative intergations already had that check.

**Comment by ishleenk17** [comment]
> I would expect some test evidence like a grok test or sth that this change worked. Apart from that is ok.

I have added the testing to the PR description.

**Pull request merged**